src/main/java/lotto/model/LottoSeller.java

```diff
@@ -3,25 +3,25 @@
 public class LottoSeller {
     private static int LOTTO_COST = 1000;
     private final int numberOfLotto;
-    public LottoSeller(String input) {
-        checkInput(input);
-        int money = parsingInteger(input);
-        checkMoney(money);
-        this.numberOfLotto = calculatorNumberOfLotto(money);
+    private final int numberOfManual;
+    private final int numberOfAuto;
+
+    public LottoSeller(Integer money, Integer manualNumber) {
+        isNullCost(money);
+        minimalMoney(money);
+        numberOfLotto = calculatorNumberOfLotto(money);
+        numberOfManual = manualNumber;
+        numberOfAuto = numberOfLotto - numberOfManual;
     }
 
-    private void checkInput(String input) {
-        if (input == null || input.isBlank()) {
-            throw new IllegalArgumentException("알맞은 숫자를 입력해주세요.");
+    private void isNullCost(Integer money) {
+        if (money == null) {
+            throw new IllegalArgumentException("알맞은 금액을 입력해주세요.");
         }
     }
 
-    private Integer parsingInteger(String input) {
-        return Integer.parseInt(input);
-    }
-
-    private void checkMoney(Integer money) {
-        if (money<1000) {
+    private void minimalMoney(Integer money) {
+        if (money < 1000) {
```

Comment:
```suggestion
        if (money < 1000) {
```

`LOTTO_COST` 매직넘버를 의미있는 이름으로 잘 도출해주셨는데 ㅜ 사용되지 않았네요 🥲